### DISCUSSION TITLE
Why doesn't ya{ and similar commands work?

### DISCUSSION THREAD

**Discussion by ragnsan** [VSCodeVim/Vim#8129] [Q&A]
I saw in a video that you can select the contents of a bracket, on the line you are on, without having to be inside the brackets using "ya{" to "select forward". 

Whenever I type ya og yi in VSCode my cursor height is cut in half like in the image below, and when I press {, i just resets the command.

What am I missing or misunderstanding?
![2022 11 30](https://user-images.githubusercontent.com/70635181/204731229-0adc6499-ea95-421e-b375-803921957066.png)

**Comment by kosekidev** [comment]
Yes ! It's the same with da{ or ca{ or va{.
Actually we need to be inside the brackets... not really intuitive I think 🤔
That would be awesome if they could implement the same behaviour than vim.
